en/snap/with-plugins.md

```diff
@@ -350,68 +399,93 @@ Customer receives email notification| Receive notification|  Check MAP menu sett
-    * Fill Payment Option Display Text with text button that you want to display to Customer.
-    * Select **Environment**, Development for testing transaction, Production for real transaction.
-    * Fill in the **Merchant ID,** **Client key & Server key**. You can find this credential on Midtrans MAP `Dashboard Settings` **->** `Access Keys`.
-    * **Map payment SUCCESS status to:** select your desired order status when payment is success.
-    * **Map payment FAILURE status to:** select your desired order status when payment is failure.
-    * **Map payment PENDING/CHALLENGE status to:** select your desired order status when payment is challenged.
-    * Other configuration are optional, you may leave it as is.
+## PrestaShop
+Midtrans ❤️ PrestaShop! Integrate your PrestaShop store with Midtrans Snap payment gateway. Midtrans strives to make payments simple for you and the customers. This plugin will allow online payment on your PrestaShop store using various online payment channels.
+
+#### Requirements
+Some of the requirements to continue with the integration process, are listed below.
+* PrestaShop 1.6 and 1.7 or later | Tested up to v1.7
+* PHP version 5.4 or later
+* MySQL version 5.0 or later
+* Midtrans plugin for PrestaShop [ [GitHub](https://github.com/veritrans/SNAP-Prestashop) | [Zip](https://github.com/veritrans/SNAP-Prestashop/archive/master.zip) ]
+
+#### PrestaShop Plugin Installation and Configuration
+To install and configure the Midtrans-PrestaShop plugin, follow the steps given below.
+1. Download the plugin file from the link given above.
+2. Extract the plugin and rename the folder as **midtranspay**. Then Zip the folder back into **midtranspay.zip**.
+3. Go to **IMPROVE > Modules > Modules Manager** on PrestaShop administration page.
+4. Click **Upload a module**.
+5. Locate the **midtranspay.zip** file, click **Open**.
+
+   The plugin is installed successfully. A message to confirm the action is displayed.
+6. Click **Configure**.
+7. Find the **Midtrans Pay** module in the module manager and click **Configure**. Configure *Midtrans Pay* page is displayed. Follow the steps given below.
+    * Enter **Payment Option Display Text**. This text appears on the button displayed to the customer.
+    * In the **Environment** list, click the appropriate environment. *Development* for testing transaction, *Production* for real transaction.
+    * Enter **Merchant ID**.
+    * Enter **Client key.**
+    * Enter **Server key**.
+    * Select desired order status for successful payments, from **Map payment SUCCESS status to** list.
+    * Select desired order status for payment failure, from **Map payment FAILURE status to** list.
+    * Select desired order status for challenge payment, from **Map payment PENDING/CHALLENGE status to** list.
+
+     ?>***Note***: Other fields are optional. You may leave it as is.
+
+The plugin is installed and configured successfully.
```

Comment:
Mind the spacing please

Reply:
The spacing between 5. and 6. are deliberate. So that the line "The plugin is installed..." appears as a separate line. The spacing above and below the "Notes" is also deliberate. Otherwise, it is not rendered properly on localhost.